docker/base/Dockerfile

```diff
@@ -44,13 +46,10 @@ echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.lis
 apt-get update && apt-get install yarn -y && \
 apt-get clean
 
-##################
-#    MiniConda   #
-##################
-RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
-    wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
-    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
-    rm ~/miniconda.sh
+##########
+#  Node  #
+##########
+RUN conda install -y -c conda-forge nodejs
```

Comment:
consolidate with the other conda install, it's best to minimize the number of Docker commands.
combine the apt-get and conda installs as much as possible.

Reply:
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices 
"Minimize the number of layers"

Reply:
GTK Thanks!